.buildkite/pipeline.yml

```diff
@@ -1,9 +1,14 @@
-use_plugins: &use_plugins
-    plugins:
-      - automattic/bash-cache#v1.0.0: ~
-      - automattic/git-s3-cache#add/vm-host-support:
-          bucket: "a8c-repo-mirrors"
-          repo: "wordpress-mobile/wordpress-ios/"
+# Nodes with values to reuse in the pipeline.
+common_params:
```

Comment:
Alternative syntaxes I tried here:

```yml
x_common_params:

<<:

_:
```

The all worked. I though of the one with the `x_` prefix because of the `x` prefix convention mentioned earlier in the discussion. I thought of `<<` and `_` to avoid having a key and tell the reader that this node was something we defined outside of the pipeline spec.

I opted not to use either `<<` or `_` because they are not that clear; if we removed the comment, the purpose of the syntax might get lost. Conversely, `common_params` and `x_common_params` are self-explanatory even without the comment.

I'm happy to use the `x_` prefix, but I wonder if we really need the extra information it carries in this configuration.

Reply:
I'm ok with not having the x prefix.

One thing that could be nice, despite this key being totally arbitrary, is to try to always use the same name for it across the various YAML and repositories, just for consistency. So we might want to bikeshed it a bit to find one that everyone is happy with. But that's really nitpicks so nbd.